Project/Source/FileSystem/AudioImporter.cpp

```diff
@@ -51,4 +66,41 @@ bool AudioImporter::ImportAudio(const char* filePath, JsonValue jMeta) {
 	unsigned timeMs = timer.Stop();
 	LOG("Audio imported in %ums", timeMs);
 	return true;
+}
+
+void AudioImporter::EncondeWavToOgg(const char* infilename, const char* outfilename) {
+	static double data[BUFFER_LEN];
+
+	SNDFILE *infile, *outfile;
+	SF_INFO sfinfo;
+	int readcount;
```

Comment:
Code Style:
- Variables should be named using camelCase:
  - inFile, outFile, inFileName...